vehicle-simulator/README.md

```diff
@@ -0,0 +1,110 @@
+![Kanto logo](https://github.com/eclipse-kanto/kanto/raw/main/logo/kanto.svg)
+
+# Eclipse Kanto - Vehicle Simulator
+
+# Introduction
+
+The simulator is used to feed simulated vehicle data and showcase the cloud connectivity that comes with Kanto. The data is really simple and simulates a small portion of actual data that can be sent by actual vehicle. This data is sent to the local mqtt broker and depending on the Kanto configuration uploaded in the corresponding cloud. It is especially designed to work with the BLING demo application. This application reads the data from the corresponding cloud instance and visualizes it. Additionally both the simulator and the demo application support sending commands from the demo application through the cloud instance, to the corresponding Kanto cloud connector and finally to the simulator.
+
+# Telemetry data
+
+The simulator sends periodic updates for a ditto feature called *OBD*. This feature contains information for the Control Module Voltage and for three standard DTCs (Device Trouble Code) b1601, b1602 and b1603. The values are randomly generated. Here is a sample ditto message.
+
+```json
+{
+   "topic":"<YourDeviceId>:Vehicle/things/twin/commands/modify",
+   "headers":{
+      "content-type":"application/json",
+      "response-required":false
+   },
+   "path":"/features/OBD",
+   "value":{
+      "properties":{
+         "controlModuleVoltage": 9,
+         "b1601": true,
+         "b1602": false,
+         "b1603": true
+      }.
+   }
+}
+```
+
+Another data sent from the simulator but only after some user interaction, which will be explained later is a ditto feature called *trunk*. It contains information if the trunk is open or closed. A sample message for the trunk state looks like this.
+
+```json
+{
+   "topic":"<YourDeviceId>:Vehicle/things/twin/commands/modify",
+   "headers":{
+      "content-type":"application/json",
+      "response-required":false
+   },
+   "path":"/features/trunk",
+   "value":{
+      "properties":{
+         "isOpen":true
+      }
+   }
+}
+```
+
+# Commands
+
+The simulator supports the following commands.
+
+- Open Trunk
+- Reset DTC
+
+## Open Trunk
+
+When Open Trunk command is received, the simulator sends a *trunk* feature update, reporting *"isOpen" : true*, simulating opening of the trunk. After predefined, but configurable amount of time, the simulator sends *trunk* feature update, reporting *"isOpen" : false*, simulating closing of the trunk.
+
+To invoke the command one needs to send a message from the cloud instance to the corresponding connected device, where the simulator is running. The exact way to do this depends on the cloud itself, but the message payload has to be in the following ditto format.
+
+```json
+{
+  "topic": "<YourDeviceId>:Vehicle/things/twin/commands/openTrunk",
+  "headers": {
+  "content-type": "application/json",
+    "response-required": false
+  },
+  "path": "/"
```

Comment:
Is this message to the thing or to the DTC feature? The path should be aligned per the case, for reference check the [Ditto spec](https://eclipse.dev/ditto/protocol-specification-things-messages.html).